### PR TITLE
Pending feedback: Add phpcs.xml to project template to help unify coding style

### PR DESCRIPTION
**Pull request by ottok**
Pending on a decision if we should ship this in the project template or not.

Related posts:
- https://seravo.com/blog/coding-wordpress-in-style-with-phpcs/
- https://seravo.com/blog/best-php-code-style-for-wordpress-developers/

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  <exclude-pattern>/htdocs/wp-content/plugins/index.php</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/mailpoet</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/maintenance</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/post-duplicator</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/redirection</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/simple-custom-post-order</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/simple-header-footer-html</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/wp-recaptcha-integration</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/caldera-forms</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/native-lazyload</exclude-pattern>
```
Comment (elguitar) [inline phpcs.xml:46]: This list only applies to one site. Every site should have their own excludes. I believe that these should be replaced with a comment explaining that the user should exclude directories that they are not developing themselves.

File: phpcs.xml [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    Exclude CodeCeption tests to prevent errors: "Variable "$I" is not in
+    valid snake_case format, try "$i"".
+  -->
+  <exclude-pattern>/codeception/</exclude-pattern>
+
+  <!-- Exclude uploads where some plugins (read MailPoet) stores PHP files -->
+  <exclude-pattern>/htdocs/wp-content/uploads/</exclude-pattern>
+
+  <!-- Don't touch Contrast's code and dependencies which are named as "libraries" -->
+  <exclude-pattern>/cntrst-*/</exclude-pattern>
```
Comment (elguitar) [inline phpcs.xml:61]: Seravo specific

**Review by elguitar — CHANGES_REQUESTED** [review]
I think that there is no harm including this to `wordpress` repo.

However, there were some seravo-specific excludes, which would be better left out.

File: phpcs.xml (hunk 1 above)
Comment (ottok) [reply phpcs.xml:46]: Updated.

**Comment by ottok** [comment]
The first version of this was just to get a round of comments on if the idea of including phpcs.xml at all is a good thing or not. Based on initial discussions, the answer is yes, so updated now the phpcs.xml file to be more like a final version.

**Review by elguitar — APPROVED** [review]
LGTM!

**Review by elguitar — APPROVED** [review]
Looks good!

**Pull request merged**